src/main/java/nl/topicus/mssql2monetdb/CopyToolConfig.java

```diff
@@ -731,6 +733,17 @@ else if (key.equals("port"))
 		
 		return sourceDatabases;
 	}
+	
+	private String readQueryFile(String filePath)
+	{
+		try {
+			return new String(Files.readAllBytes(Paths.get(filePath)));
+		} catch (IOException e) {
+			LOG.error("Unable to read query file {} due to: {}", filePath, e.getMessage());
+		}
+		
+		return null;
```

Comment:
Er zit later in de code een check of er voor een bepaalde tabel configuratie een tabel naam of query is opgegeven. Indien deze mist gaat de tool wel door maar zonder deze specifieke tabel configuratie. Het teruggeven van een `null` is dus prima. Ik heb in ieder geval wat JavaDoc toegevoegd.